lib/util.rb

```diff
@@ -50,8 +50,8 @@ def read
       @hash = JSON.parse(File.read(@path))
       return JSON.parse(File.read(@path))
```

Comment:
When you run `cache.read` you expect your hash to be updated, but this method should also return the parsed JSON. I can de-duplicate that `JSON.parse`.